packages/google_maps_flutter/google_maps_flutter_web/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 * Updates minimum supported SDK version to Flutter 3.22/Dart 3.4.
```

Comment:
Please see https://github.com/flutter/flutter/blob/master/docs/ecosystem/contributing/README.md#updating-a-changelog-that-has-a-next

This is why `repo_checks` is failing.